crates/constants/src/lib.rs

```diff
@@ -14,7 +14,7 @@ pub const HULA_DBUS_SERVICE: &str = "org.hulks.hula";
 pub const HULA_SOCKET_PATH: &str = "/tmp/hula";
 pub const OS_RELEASE_PATH: &str = "/etc/os-release";
 pub const OS_VERSION: &str = "7.4.2";
-pub const SDK_VERSION: &str = "7.4.0";
+pub const SDK_VERSION: &str = "7.4.2";
```

Comment:
Why was this done? This leads to all HULKs downloading the newest patch version when running pepsi?! And using bumping this without also bumping the pepsi version also doesn't make sense... Please explain ;)

Reply:
This was necessary after updating the CI image. Without it, our CI couldn't find the SDK (it was looking for version 7.4.2, the latest meta-nao release). Knoelle knows the details of this.